metriq_gym/job_manager.py

```diff
@@ -0,0 +1,59 @@
+import json
+import uuid
+from typing import List, Dict, Any
+
+DEFAULT_FILE_PATH = ".metric_gym_jobs.jsonl"
+
+# TODO: JobManager is not thread-safe at the moment
+
+
+class JobManager:
+    def __init__(self, file_path: str = DEFAULT_FILE_PATH):
+        self.file_path = file_path
+        self._load_jobs()
+
+    def _load_jobs(self):
+        self.jobs = []
+        try:
+            with open(self.file_path, "r") as file:
+                for line in file:
+                    try:
+                        job = json.loads(line.strip())
+                        self.jobs.append(job)
+                    except json.JSONDecodeError:
+                        continue
+        except FileNotFoundError:
+            self.jobs = []
```

Comment:
Since `self.jobs = []` is set at the beginning of this function, is it not redundant to have the exception also set this to an empty list?